.gitignore

```diff
@@ -10,3 +10,4 @@ __pycache__
 /data
 .coverage
 .pytest_cache/
+.vscode/
```

Comment:
FYI, you can use a [host-level gitignore file](https://git-scm.com/docs/gitignore#_synopsis) under `XDG_CONFIG_HOME` (~/.config/git/ignore on linux) so you don't have to add editor-specific ignores to every repo.